docs/concepts/privacy-and-security.md

```diff
@@ -251,9 +251,9 @@ The management and enforcement of Office settings is done with group policy sett
 
 | Setting name | Description |
 |--------------|-------------|
-| Allow Unsecure web add-ins and Catalogs | Allows users to run non-secure add-ins, which are add-ins that have webpage or catalog locations that are not SSL-secured (https://) and are not in users' Internet zones. |
-| Block Web Add-ins | Allows you to prevent users from using web add-ins. |
-| Block the Office Store |  Allows you to prevent users from using or inserting web add-ins that come from the Office Store. |
+| Allow Unsecure web add-ins and Catalogs | Allows users to run non-secure Office Add-ins, which are Office Add-ins that have webpage or catalog locations that are not SSL-secured (https://) and are not in users' Internet zones. |
+| Block Web Add-ins | Allows you to prevent users from running Office Add-ins using web technologies. |
+| Block the Office Store |  Allows you to prevent users from running or inserting Office Add-ins that come from the Office Store. |
```

Comment:
The button in the ribbon is called **Get Add-ins**, so mybe using "get" instead of "insert" is more intuitive?
```suggestion
| Block the Office Store |  Allows you to prevent users from getting or running Office Add-ins that come from the Office Store. |
```

Reply:
Thanks! These are great suggestions.